### PR TITLE
Add running tests to README.md

### PR DESCRIPTION
**Pull request by iblancasa**
## Which problem is this PR solving?
- Require running the tests as one of the steps to create a release.

### CODE REVIEW

**Review by frzifus — APPROVED** [review]
nice :)

File: RELEASE.md [hunk 1]
```diff
@@ -14,29 +14,31 @@
...
+2. Run `OPERATOR_VERSION=1.30.0 make prepare-release`, using the operator version that will be released.
 
-1. Prepare a changelog since last release.
+3. Run the E2E tests in OpenShift as described in [the CONTRIBUTING.md](CONTRIBUTING.md#an-external-cluster-like-openshift) file. The tests will be executed automatically in Kubernetes by the GitHub Actions CI later.
 
-1. Commit the changes and create a pull request:
+4. Prepare a changelog since last release.
 
-   ```
+5. Commit the changes and create a pull request:
```
Comment (rubenvp8510) [inline RELEASE.md:25]: I think the intention of have 1.  in each bullet is to don't have to re-enumerate if we add more :)

Comment (iblancasa) [reply RELEASE.md:25]: Do you want to recover that or is OK with the current approach?

Anyway, if you do this:
```markdown
1.
5.
3.
```

Markdown is rendered as:

1.
5.
3.

But is more legible to follow the steps if you are just reading the MD file

Comment (rubenvp8510) [reply RELEASE.md:25]: ha hhaha unexpected to me, ok then it is ok.

**Pull request merged**